libraries/botframework-connector/src/auth/authenticateRequestResult.ts

```diff
@@ -24,5 +24,5 @@ export type AuthenticateRequestResult = {
     /**
      * A value for the ConnectorFactory.
      */
-    connectorFactory: ConnectorFactory;
+    connectorFactory?: ConnectorFactory;
```

Comment:
Made optional to follow the .NET implementation where [AuthenticateStreamingRequestAsync](https://github.com/microsoft/botbuilder-dotnet/blob/ececa515fc05c331d4ceeeca4b2b031fee39958c/libraries/Microsoft.Bot.Connector/Authentication/ParameterizedBotFrameworkAuthentication.cs#L100) doesn't set a ConnectorFactory member on the returned `AuthenticateRequestResult`.